src/js/theme.js

```diff
@@ -102,7 +102,7 @@ class Theme {
               document.body.setAttribute("theme", "light");
             else document.body.setAttribute("theme", "dark");
             this.isDark = !this.isDark;
-            window.localStorage &&
+            this.config.cookieconsent.enable && window.localStorage &&
```

Comment:
If the `cookieconsent` will be disabled from the config, the entire object will be `undefined` in js, so in this case, the `Uncaught TypeError: Cannot read property 'enabled' of undefined'` error will be received.

The requirements here are to check the `cookieconsent` object against `undefined`.

CC @aplatanado